bin/countperl

```diff
@@ -482,6 +490,72 @@ sub get_class_by_count {
 
 ### /HTML
 
+### JSON
```

Comment:
I should consider moving the JSON (and HTML) stuff into modules in `lib` so the functions are available outside of `countperl`, or maybe make `countperl` testable, in either case I could add some test coverage.